public/app/features/browse-dashboards/BrowseDashboardsPage.tsx

```diff
@@ -68,9 +83,15 @@ const BrowseDashboardsPage = memo(({ match }: Props) => {
           <AutoSizer>
             {({ width, height }) =>
               isSearching ? (
-                <SearchView key={rerender} width={width} height={height} />
+                <SearchView key={rerender} showCheckboxes={canEditInFolder} width={width} height={height} />
```

Comment:
lets rename this to `canManage` or `canSelect` something. We want to describe capabilities, not figure implementation details.

Reply:
@joshhunt Renaming `showCheckboxes` I assume?

Reply:
Sorry, yes 👍🏻